conda_smithy/lint_recipe.py

```diff
@@ -723,7 +723,7 @@ def run_conda_forge_specific(meta, recipe_dir, lints, hints):
             # It's a team. Checking for existence is expensive. Skip for now
             continue
         try:
-            gh.get_user(maintainer)
+            gh.get_user({0}.format(maintainer))
```

Comment:
maybe just?
```suggestion
            gh.get_user(str(maintainer))
```

Can you create a test for it please? is it possible?

Reply:
Well, you can monkeypatch the git process which return the user right? monkey patch it to return an hexadecimal
In that way you will reproduce the problem